<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_okayforreg">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancelforreg">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>261</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_stname">
      <property name="text">
       <string>Student Name  (Also login id)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_studentnameforreg"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Student ID</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_studnetidforreg"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Login password</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_passwordforreg"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Numbers of Courses Registed</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_numbersofcourses"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>261</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Courses enrolled</string>
   </property>
   <widget class="QCheckBox" name="boxCS3306">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>73</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CS3306</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">courses_button</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="boxCS3321">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>73</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CS3321</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">courses_button</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="boxCS4108">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>73</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CS4108</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">courses_button</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="boxCS4109">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>73</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CS4109</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">courses_button</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="courses_button">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
